driver/lib/sessions/ios.ts

```diff
@@ -35,10 +32,18 @@ export const getObservatoryWsUri = async (proxydriver) => {
   if (realDevice) {
     // @todo check if `brew install usbmuxd` is needed
     log.info(`Running on iOS real device, doing "iproxy" now`);
-    const cmd = `iproxy ${urlObject.port} ${urlObject.port} ${udid}`;
-    log.debug(cmd);
-    await execPromise(cmd);
-    log.info(`"iproxy" successfully`);
+    const args = [urlObject.port, urlObject.port, udid];
+    log.debug(`iproxy ${urlObject.port} ${urlObject.port} ${udid}`);
+    const cmd = spawn(`iproxy`, args);
+    let stdout = ``;
+    cmd.stdout.on(`data`, (data) => {
```

Comment:
This event never gets triggered because of this [issue](https://github.com/libimobiledevice/libusbmuxd/issues/79).  I made another [issue](https://github.com/libimobiledevice/libusbmuxd/issues/103) to follow up with.

Reply:
We might add {shell:true} to get the logs. Could you please try to build this locally with this am trying to avoid hard code wait here

Reply:
I tested `{shell:true}` but it didn't work.